MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.rst AUTHORS LICENSE CHANGES.rst MANIFEST.in
 graft src
 recursive-exclude src *.pyc
 recursive-exclude src *.pyo
+recursive-exclude src *.so
```

Comment:
I only did the minimal required change here.

More work should be done in 1.0-maint, then merged into master.